### PR TITLE
Add Amazon Web Services Alternatives

### PR DESCRIPTION
**Pull request by ro31337**
https://github.com/ro31337/awesome-aws-alternatives

This is the list of AWS alternatives. Some companies are dependent too much on AWS, and it's nice to know alternatives that can be free or less expensive. IMO worth adding to awesome list.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -85,6 +85,7 @@
 - [JVM](https://github.com/deephacks/awesome-jvm)
 - [Salesforce](https://github.com/mailtoharshit/awesome-salesforce)
 - [Amazon Web Services](https://github.com/donnemartin/awesome-aws)
+- [Amazon Web Services Alternatives](https://github.com/ro31337/awesome-aws-alternatives)
```
Comment (aschrijver) [inline readme.md]: You did not add your entry to the bottom of the list as per the [guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md#adding-to-this-list).

**Comment by aschrijver** [comment]
Checked the list and your PR:
- Your PR misses the checklist (check other PR's)

Various things are missing or incorrect:
- Table of Contents missing
- Some entries have description, others have not
- All list entries should end with a . (dot)

I would advice going through the checklist and either improve the PR or resubmit it.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
@ro31337 ⬆️

**Comment by sindresorhus** [comment]
Closing for lack of response from the submitter.

**Pull request closed without merging**